code/run_capsule.py

```diff
@@ -1,7 +1,48 @@
-""" top level run script """
+"""top level run script"""
+
+import logging
+import shutil
+from pathlib import Path
+
+from aind_nwb_utils.utils import combine_nwb_file
+from hdmf_zarr import NWBZarrIO
+from pydantic_settings import BaseSettings
+from pynwb import NWBHDF5IO
+
+
+class NWBCombineSettings(
+    BaseSettings, cli_parse_args=True, cli_ignore_unknown_args=True
+):
+    """Settings for NWBCombine"""
+
```

Comment:
can you set up an app panel with corresponding argparser for this?

Reply:
Done